20_Getting_Started/20_Build/10_Linux.md

```diff
@@ -102,6 +102,8 @@ as follows:
     -   master
         -   src/
         -   build/
+            - debug/
+            - release/
```

Comment:
Build directories may correspond to different (specific) needs, that's why they were not specified in the architecture proposal.
e.g. "Release_VS2013_Ninja_x64", "Debug_VS2013_Make_x64", "Release_GCC_Make_x32"